bootstrap.sh

```diff
@@ -389,6 +389,8 @@ get_shell_startup_script() {
 install_sentry_cli() {
   log "Installing sentry-cli"
   if ! command -v sentry-cli &>/dev/null; then
+    # This ensures that sentry-cli has a directory to install under
+    [ ! -d /usr/local/bin ] && sudo_askpass mkdir /usr/local/bin
```

Comment:
Are we going to run into permission issues here since `root` will own this dir?

Reply:
I have not had any issues. I can follow up with another fix if I find one.

Reply:
<img width="1166" alt="image" src="https://user-images.githubusercontent.com/44410/142011178-f25a371d-67c5-4451-af24-cb5653ada9cc.png">